src/NuGetGallery.Services/Configuration/IFeatureFlagService.cs

```diff
@@ -296,5 +296,10 @@ public interface IFeatureFlagService
         /// Whether or not to compute backend operations for target framework. This flag is overridden by <see cref="IsDisplayTargetFrameworkEnabled"/> if that flag is true.
         /// </summary>
         bool IsComputeTargetFrameworkEnabled();
+
+        /// <summary>
+        /// Whether or not recent packages has no index applied.
```

Comment:
Consider adding a tiny bit more detail for clarity:

```suggestion
        /// Whether or not recent packages has no index applied to block search engine indexing.
```

I don't feel strongly, feel free to keep it as is if you prefer